toolchain/check/handle_literal.cpp

```diff
@@ -73,21 +83,49 @@ auto HandleBoolTypeLiteral(Context& context, Parse::BoolTypeLiteralId node_id)
   return true;
 }
 
+static auto HandleIntOrUnsignedIntTypeLiteral(Context& context,
```

Comment:
Is it worth remarking on the semantics of this?

Reply:
Sure.
```suggestion
// Shared implementation for handling `iN` and `uN` literals.
static auto HandleIntOrUnsignedIntTypeLiteral(Context& context,
```